### DISCUSSION TITLE
New tab arrow on the “Web Search” object under the logo Firefox logo

### DISCUSSION THREAD

**Discussion by wis3f1sh** [black7375/Firefox-UI-Fix#694] [Q&A] [answered]
How i can remove this arrow from the search bar, and return the previous function (when you click on the search bar and start to write, it immediately returns to the address bar)?
![image](https://user-images.githubusercontent.com/97912016/234488598-407c81a1-f405-4343-bbf6-8a1c75bb4de8.jpeg)

**Comment by black7375** [comment, marked as answer]
1. It's need a [edit user.js](https://github.com/black7375/Firefox-UI-Fix/wiki/Tutorial#edit-userjs-file)
2. Open `user.js`
3. Change `browser.newtabpage.activity-stream.improvesearch.handoffToAwesomebar` to `true`
